test/testdata/lsp/document_symbols_minitest.rb.document-symbols.exp

```diff
@@ -28,6 +28,32 @@
                 }
             },
             "children": [
+                {
+                    "name": "<before>",
```

Comment:
I think we want to hide the existence of `<before>` from document symbols.  Before, having `initialize` there was sort of understandable, but this PR changes that to be potentially confusing.

OTOH, maybe people would like seeing the `before` block as something obvious?  Maybe instead of filtering it, we should just give it a better name?  WDYT?

Reply:
I don't have a super strong opinion here.

It's worth noting that we already surface all the `it` blocks as fake methods where the string passed to the `it` block is in the method definition. So I think that the document outline in test files is already pretty weird, in that there's way more entries in the document symbols output than there are method definitions in the file.

But since you asked, I've written the code to skip it for now.

Reply:
> It's worth noting that we already surface all the `it` blocks as fake methods where the string passed to the `it` block is in the method definition. So I think that the document outline in test files is already pretty weird, in that there's way more entries in the document symbols output than there are method definitions in the file.

I think this is a happy consequence of the rewriter and convenient for the user, but I can understand how my happiness is not exactly consistent with my request earlier. 😅

Reply:
> a happy consequence of the rewriter

How much of Sorbet's success boils to this phrase